desktop/src/com/frostwire/gui/updates/UpdateManager.java

```diff
@@ -210,16 +212,16 @@ private void handlePossibleUpdateMessage(UpdateMessageReader umr, boolean force)
 
             // Logic for Windows or Mac Update
             if (OSUtils.isWindows() || OSUtils.isMacOSX()) {
-                if (hasUrl && !hasTorrent && !hasInstallerUrl) {
+                if ((hasUrl && !hasTorrent && !hasInstallerUrl) || hasUrl && !canUseTorrent()) {
```

Comment:
let's please remove this check for the update manager. This one torrent is a system torrent, should be fine.